tests/e2e/config/setup-test-framework.js

```diff
@@ -0,0 +1,223 @@
+/**
+ * Forked from Gutenberg, with only minor changes.
+ *
+ * @see https://github.com/WordPress/gutenberg/blob/df2b096185818f89b2c1668d22a848990acb2799/packages/e2e-tests/config/setup-test-framework.js
```

Comment:
The main difference between this file and Gutenberg is the deletion of the [Axe accessibility tests](https://github.com/WordPress/gutenberg/blob/df2b096185818f89b2c1668d22a848990acb2799/packages/e2e-tests/config/setup-test-framework.js#L219). Of course, accessibility is important. But I think these tests could make this more flaky.

It looks like the tests [are still in development](https://github.com/WordPress/gutenberg/blob/df2b096185818f89b2c1668d22a848990acb2799/packages/e2e-tests/config/setup-test-framework.js#L225-L226). And they really tests accessibility of the entire block editor, not just Atomic Blocks.